src/main/java/com/venafi/vcert/sdk/connectors/tpp/TppTokenConnector.java

```diff
@@ -0,0 +1,431 @@
+package com.venafi.vcert.sdk.connectors.tpp;
+
+import com.venafi.vcert.sdk.VCertException;
+import com.venafi.vcert.sdk.certificate.*;
+import com.venafi.vcert.sdk.connectors.*;
+import com.venafi.vcert.sdk.endpoint.Authentication;
+import com.venafi.vcert.sdk.endpoint.ConnectorType;
+import com.venafi.vcert.sdk.utils.Is;
+import feign.FeignException;
+import feign.Response;
+
+import java.net.InetAddress;
+import java.text.MessageFormat;
+import java.time.Instant;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.String.format;
+import static java.time.Duration.ZERO;
+import static java.util.Objects.isNull;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+public class TppTokenConnector extends AbstractTppConnector implements TokenConnector {
+
+    public TppTokenConnector(Tpp tpp){ super(tpp); }
+
+    @Override
+    public ConnectorType getType() {
+        return ConnectorType.TPP_TOKEN;
+    }
+
+    @Override
+    public void setBaseUrl(String url) throws VCertException {
+        throw new UnsupportedOperationException("Method not yet implemented");
+    }
+
+    @Override
+    public void setZone(String zone) {
+        this.zone = zone;
+    }
+
+    @Override
+    public void setVendorAndProductName(String vendorAndProductName) {
+        this.vendorAndProductName = vendorAndProductName;
+    }
+
+    @Override
+    public String getVendorAndProductName() {
```

Comment:
nit: you don't need this since you have @Getter in the parrent

Reply:
The getVendorAndProductName() method is required by contract of the interface. Instead, the @gettter annotation was removed from the variable, as it is not being used anywhere in the code, or tests.